front/components/items/Button.tsx

```diff
@@ -3,7 +3,7 @@ import Link from 'next/link'
 import React from 'react'
 import { HeadlineType } from 'types/items/headline'
 import Headline from './Headline'
-const Button = ({h1, h2, h3, h4, h5, h6, text, color="green", place="center", link, isUppercase, isLink, isImage, className, image}: ButtonType) => {
+const Button = ({h1, h2, h3, h4, h5, h6, text, color="green", place="center", link, isUppercase, isLink, isImage, className, image, type}: ButtonType) => {
```

Comment:
Idk what those parameters mean, but it's not going right if you have to pass h1..h6 as a props

Reply:
I created seperate component for headlines, so there's a need to use all parameters to have ability to choose which size I wanna use. In that case I just pass the props from headline into button component